GliaWidgets/Component/Badge/BadgeStyle.swift

```diff
@@ -26,4 +26,29 @@ public struct BadgeStyle {
         self.fontColor = fontColor
         self.backgroundColor = backgroundColor
     }
+
+    /// Apply badge remote configuration
+    public mutating func applyBadgeConfiguration(_ badge: RemoteConfiguration.BadgeStyle?) {
+        if let font = UIFont.convertToFont(font: badge?.font) {
+            self.font = font
+        }
+
+        badge?.fontColor?.value.map { colors in
```

Comment:
JFYI, you can simplify it. Anyway we will change color usage to handle gradients.
```suggestion
        badge?.fontColor?.value.map { fontColor = UIColor(hex: $0[0]) }
```

Reply:
Fixed! Good top!